### PR TITLE
Fix deep learning terms

### LINKED ISSUE #576: 2023-03-08: English/Deep learning terms
I am currently working on the glossary for the [deep learning lesson](https://github.com/carpentries-incubator/deep-learning-intro).

There are a couple of terms that I need that are currently missing from glosario:

- artificial intelligence
- convolutional neural network (CNN)
- recurrent neural network (RNN)
- epoch
- learning rate
- class imbalance
- hidden layers

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #576 (hopefully)

### CODE REVIEW

**Comment by elletjies** [comment]
@all-contributors please add @svenvanderburg for translation
